driver/compenv.ml

```diff
@@ -818,8 +714,15 @@ let process_deferred_actions env =
     end
   else if !deferred_actions = [] then
     fatal "No input files";
-  List.iter (process_action env) (List.rev !deferred_actions);
->>>>>>> 5.2.0
+  (* XXX mshinwell: do we need this? *)
```

Comment:
(Just noting I did not assess this question as part of review - let me know if you expected me to.)